lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +18,15 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (isinstance(text, str) and text):
+        return None
+    punctuation = '''.,;':¡!¿?…⋯‹›«»\\/"“”[]()⟨⟩}{&|-–~—'''
+    for punc in punctuation:
+        text = text.replace(punc, ',')
+    phrase_list = text.split(',')
+    phrase_list = [phrase.strip() for phrase in phrase_list if phrase.strip()]
```

Comment:
return directly this list. use construction from https://www.youtube.com/watch?v=ZBv-G4vxxyc